src/widget-sources/types.ts

```diff
@@ -59,6 +59,8 @@ export interface CategoryRequestOptions extends BaseRequestOptions {
   operationColumn?: string;
   /** Local only. */
   joinOperation?: 'count' | 'avg' | 'min' | 'max' | 'sum';
+  /** Calculate `_carto_others` category for all categories after first N (N is threshold). Remote only. */
+  othersThreshold?: boolean;
```

Comment:
If `othersThreshold` is a `boolean`, how can we specify the number `N` for telling the Backend at which **threshold** we should calculate the "rest"?